test/utils/InstanceTest.sol

```diff
@@ -463,7 +522,16 @@ abstract contract InstanceTest is HyperdriveTest {
         }
 
         // Contribution in terms of base.
-        uint256 contribution = 1_000e18;
+        uint256 contribution;
+        if (!isBaseETH) {
```

Comment:
maybe add some comments around branches of this if statement (or describe in block comment)